app/src/main/res/values/strings.xml

```diff
@@ -388,4 +388,10 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_parkingType_surface">Normal</string>
     <string name="quest_parkingType_underground">Underground garage</string>
     <string name="quest_parkingType_multiStorage">Car park</string>
+    <string name="quest_dietType_vegetarian_name_title">Does "%s" explicitly offer vegetarian food?</string>
```

Comment:
I have a problem with "explicitly" here. `quest_dietType_explanation` may hopefully be sufficient, but with this "explicitly" I would not tag a restaurant offering 2-4 meals as "vegetarian". Is that intended? Would not 2-4 meals be enough for a restaurant?

Reply:
I understood "explicitly" as meaning that vegetarian meals are on the menu and you don't need to ask for them (e.g. "Can I have this but without the meat, please?").

Reply:
Ah, that's a different way I understood it. Maybe change it to "offer on it's menu"?